### PR TITLE
#36899 fill in document metadata on process link

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ Document. A reference to the document has to be available as a process variable.
 document, the resulting document URL will be saved in a process variable.
 
 When creating a process link the following properties have to be entered:
+- **File name** The name of the physical file in which the content of the document is captured, including extension.
```
Comment (marijnritense) [inline using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md:42]: Missing a period after each of the bold texts.

Comment (ivo-ritense) [reply using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md:42]: What do you mean? Should it be 
```
**File name**. The name of the physical....
```?

The way it is now is in line with the other plugin pages though.

Comment (Klaas-Ritense) [reply using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md]: I added a . to all pages

Comment (ivo-ritense) [reply using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md]: Honestly, i don't really like the way it looks:

image

**Review by marijnritense — APPROVED** [review]

File: using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md (hunk 1 above)
Comment (marijnritense) [reply using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md]: I like it more than without the dot.

Comment (Klaas-Ritense) [reply using-valtimo/plugin/documenten-api/configure-documenten-api-plugin.md]: I dont mind either way

**Pull request merged**